content/docs/reference/_index.md

```diff
@@ -2,11 +2,9 @@
 title: Reference
 ---
 
-This section provides all published technical documentation for the Pulumi platform.
+This section provides all published technical documentation for the Pulumi platform. Refer to the index below to help navigate around the content.
```

Comment:
Looking at the page, it's not clear what `the index below` refers to actually. Assuming this means "the links on the page", maybe this isn't needed?